src/components/BusinessDonationHistory/BusinessDonationHistory.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+
+import { useEffect, useState } from 'react';
+import { useBackend } from '../../contexts/BackendContext';
+import { Table, Thead, Tbody, Tr, Td, Button, Th } from '@chakra-ui/react';
+import ViewDonationHistory from './ViewDonationHistory/ViewDonationHistory';
+
+const BusinessDonationHistory = () => {
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+  const [selectedDonationId, setSelectedDonationId] = useState(null);
+  const TABLE_HEADERS = ['Submitted By', 'Food Bank', 'Date', 'Action'];
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const response = await backend.get('/donation');
+        setData(response.data);
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'backend'. Either include it or remove the dependency array.